rfc/rfc004-package-log.md

```diff
@@ -12,8 +12,21 @@ http://peter.bourgon.org/go-kit/#package-log
 
 ## Scope
 
-To be defined.
+* Log SHALL be drop in compatible with the standard library logger.
+* Log SHALL use key / value pairs for structured data.
```

Comment:
I'd love type safety to be preserved in the structured logging front. For the lack of better mechanisms, code generation + go generate could come to the rescue, receiving users' types as input. Just an idea :-)